SimPEG/Directives.py

```diff
@@ -303,26 +303,27 @@ def endIter(self):
         # Set the weighting matrix to None so that it is recomputed next time
         # it is called in the inversion
         self.reg._W = None
+        self.reg._Wsmooth = None
```

Comment:
Needed to update the def W to include the list of gradients for the inversion to run. Testing the directive now.